packages/manager/modules/pci/src/projects/project/instances/instance/rescue/rescue.html

```diff
@@ -1,34 +1,58 @@
 <form novalidate name="rescueForm">
     <oui-modal
         data-heading="{{ 'pci_projects_project_instances_instance_rescue_title' | translate }}"
-        data-primary-action="$ctrl.rescueInstance()"
-        data-primary-label="{{:: 'pci_projects_project_instances_instance_rescue_submit_label' | translate }}"
+        data-primary-action="$ctrl.hasRescueMode ? $ctrl.rescueInstance() : $ctrl.goBack()"
```

Comment:
**quetsion:** If user cannot perform any action on such instance, why cant we just hide the modal?

Reply:
The modal is used to display the information that on the Metal instances it is different and that you have to follow the documentation.